pinot-broker/src/main/java/org/apache/pinot/broker/routing/instanceselector/InstanceSelectorFactory.java

```diff
@@ -40,44 +43,48 @@ private InstanceSelectorFactory() {
   public static final String LEGACY_REPLICA_GROUP_OFFLINE_ROUTING = "PartitionAwareOffline";
   public static final String LEGACY_REPLICA_GROUP_REALTIME_ROUTING = "PartitionAwareRealtime";
 
+  @VisibleForTesting
   public static InstanceSelector getInstanceSelector(TableConfig tableConfig,
-      ZkHelixPropertyStore<ZNRecord> propertyStore, BrokerMetrics brokerMetrics) {
-    return getInstanceSelector(tableConfig, propertyStore, brokerMetrics, null, Clock.systemUTC());
+      ZkHelixPropertyStore<ZNRecord> propertyStore, BrokerMetrics brokerMetrics, PinotConfiguration config) {
```

Comment:
(minor) Suggest renaming to `brokerConf` to make it clear that it is top level broker config

Reply:
Ack